### ISSUE TITLE
Error on 4th step

### ISSUE THREAD

**Issue by YuriGagarine** [liperium/Linux-Conservation-Mode-Utility#1]
After following each step, the forth one trig an error : 

`[16658] Error loading Python lib '/tmp/_MEIJKnBlL/libpython3.9.so.1.0': dlopen: /lib/x86_64-linux-gnu/libc.so.6: version `GLIBC_2.33' not found (required by /tmp/_MEIJKnBlL/libpython3.9.so.1.0)`

I just executed the TrayConservationMode with the following command: 
`./TrayConservationMode`

**Comment by liperium** [comment]
I will look it up. Haven't played in this project in a long time. I don't really understand as to why it's looking for a pythonlib? I re-written this version of the project in go. I don't know why you are having this problem! Looking into it right now!

**Comment by YuriGagarine** [comment]
If that helps, I am on Kubuntu 20.04.05 LTS with Kernel: Linux 5.15.0-46-generic.

Ok wrong release. I downloaded the one built from Go and I have this error:

`./TrayConservationMode: error while loading shared libraries: libayatana-appindicator3.so.1: cannot open shared object file: No such file or directory`

Is it something you understand ?

**Comment by YuriGagarine** [comment]
I did some progress about the issue. Actually the problem is from the glibc version. The release that you built was compiled with glibc 2.34 but on kubuntu 20.04 the last glibc version is 2.31.

In order to fix the issue, I need you tell me how to compile your code. Indeed I never experience Go in my life, could you please describe how to get the TrayConservationMode executable ?

After that I will put a Pull Request in order to update the README for people with earlier ubuntu release.

Thanks

**Comment by YuriGagarine** [comment]
I finally make it work. I had to build everything.

**Comment by liperium** [comment]
I'm sorry for not responding or finding a fix. My lenovo is currently running Fedora and I can't make it compile as the systray library needs libayatana-qppindicators3-0.1, which is not available anywhere as an rpm package... I don't know how I'll be able to revise this application anymore, but I'm thinking about switching libraries and seeing if it's drastically bigger or not... 
Hope this serves you well! Have a good one :)

**Comment by liperium** [comment]
Hope you're enjoying the app! I have made a couple of tweaks to the structure if you want to update. But there's no real feature addition ( other than being able to do the commands from the command line! )

**Comment by YuriGagarine** [comment]
Thank you for the update ! Your app is a killer ! Do you know if the battery conservation mode is still on when the laptop is shutdown ?

**Comment by liperium** [comment]
So, what I understand is, if nothing else has tampered with it ( from my experience no linux program tampers with it ). It should keep the same state as you set it before. 
Though, for me, on Windows, sometimes it gets reset. 
I was thinking about adding a default value, or like a check if the value has changed and setting it back to the old one on startup.

TLDR : Except known Windows resets, nothing else changes that file/function.
